scala-package/core/src/main/scala/org/apache/mxnet/NDArray.scala

```diff
@@ -197,6 +197,15 @@ object NDArray extends NDArrayBase {
       "_onehot_encode", Seq(indices, out), Map("out" -> out))(0)
   }
 
+  /**
+    * Get the String representation of NDArray
+    * @param nd input NDArray
+    * @return String
+    */
+  def toString(nd : NDArray) : String = {
+    nd.visualize
```

Comment:
I want to test this out through the Clojure api. We do a lot of work in the REPL and with a default visualize in the .toString method I want to make sure the usability is not surprising, (especially with large arrays).

I'll take a look and let you know